client/src/index.ts

```
@@ -0,0 +1,46 @@
+import 'dotenv/config';
+import { drizzle } from 'drizzle-orm/node-postgres';
+const db = drizzle(process.env.DATABASE_URL!);
+
+import 'dotenv/config';
+import { drizzle } from 'drizzle-orm/node-postgres';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'drizzle'. Consider to delete it or rename it.

'drizzle' is defined here:


(lint/suspicious/noRedeclare)